yarn-project/ivc-integration/src/native_client_ivc_integration.test.ts

```diff
@@ -27,6 +27,7 @@ describe('Client IVC Integration', () => {
   // 1. Run a mock app that creates two commitments
   // 2. Run the init kernel to process the app run
   // 3. Run the tail kernel to finish the client IVC chain.
+  // 4. Run the hiding kernel.
   it('Should generate a verifiable client IVC proof from a simple mock tx', async () => {
     const [bytecodes, witnessStack, _, vks] = await generate3FunctionTestingIVCStack();
```

Comment:
Can you rename these test methods from generate3FunctionTestingIVCStack/generate6FunctionTestingIVCStack to something more general